python/observation-publisher/publisher/config.py

```diff
@@ -12,20 +12,24 @@ class ArizeConfig:
     space_key: str
 
 
-@unique
 class ObservabilityBackendType(Enum):
```

Comment:
As much as i love StrEnum, it is only on Python 3.11. I do think we should use string instead of int, so that the value doesn't change when the order change.